writer/writer/core/writer_service.py

```diff
@@ -33,20 +33,31 @@ def write(self, write_request: WriteRequest) -> None:
             with self.database.get_context():
                 self.write_with_database_context()
 
-            stats: Dict[str, int] = defaultdict(int)
-            for event in write_request.events:
-                log_name = (
-                    type(event)
-                    .__name__.replace("Request", "")
-                    .replace("Event", "")
-                    .upper()
-                )
-                stats[log_name] += 1
-            stats_string = ", ".join(f"{cnt} {name}" for name, cnt in stats.items())
-            logger.info(f"Events executed ({stats_string})")
-
             # Only propagate updates to redis after the transaction has finished
-            self.messaging.handle_events(self.db_events, self.position)
+            self.messaging.handle_events(self.db_events, self.position, log=log)
+
+            self.print_stats()
```

Comment:
@jsangmeister Either add `if log` here, too, or rename it to `log_all_modified_fields`

Reply:
Done